### PR TITLE
Update names for desc. rep metric

### PR DESCRIPTION
**Pull request by jwalsh28**

### CODE REVIEW

File: 05_local-governance/descriptive-representation/descriptive_representation_denom.qmd [hunk 1]
```diff
@@ -341,32 +341,32 @@ Keep only relevant variables before export.
 
 ```{r}
 county_pop_by_race <- county_pop_by_race %>% 
-  select(year, state, county, asian_other, black_nonhispanic, hispanic = total_hispanic, white_nonhispanic,
-         total_nonhispanic_quality = total_nonhisp_quality, asian_other_quality, 
-         black_nonhispanic_quality, total_hispanic_quality, white_nonhispanic_quality) %>%
+  select(year, state, county, other = asian_other, black_nonhispanic, hispanic = total_hispanic, white_nonhispanic,
+         nonhispanic_quality = total_nonhisp_quality, other_quality = asian_other_quality,
```
Comment (cdsolari) [inline 05_local-governance/descriptive-representation/descriptive_representation_denom.qmd:345]: We do not have a variable for share_desc_rep_nonhispanic_quality. We only have 4 categories in our metrics data tables: black_nh, white_nh, hispanic, other. So, in the final dataset, we do not need to retain that variable. I believe you need to use it to calculate the other category. It is called "Other races and ethnicities." To me, it looks like you still have one too many _quality variables. It should be 4 main variables and 4 quality variables. Sorry, can you take one more pass? Thank you!!!

**Review by cdsolari — CHANGES_REQUESTED** [review]
I think you still have one loitering _quality variable to remove from the final data: share_desc_rep_nonhispanic_quality. Thanks!

**Comment by jwalsh28** [comment]
Good catch! Just updated @cdsolari

**Review by cdsolari — APPROVED** [review]
Looks good!

**Pull request merged**